index.js

```diff
@@ -53,8 +54,18 @@ const DEFAULTS = {
 };
 
 module.exports = function(options) {
+  if (isFunction(options)) {
```

Comment:
In general, I tend to prefer readability over convenience. But since this is not used anywhere else in the module, I guess I could take the dependency down and just go along with `typeof`.